src/bbs2gh/Services/BbsSshArchiveDownloader.cs

```diff
@@ -5,7 +5,7 @@
 
 namespace OctoshiftCLI.BbsToGithub.Services;
 
-public class BbsArchiveDownloader : IDisposable
+public sealed class BbsSshArchiveDownloader : IBbsArchiveDownloader, IDisposable
```

Comment:
why is this sealed?

Reply:
for two reasons:
1. Simpler disposable pattern. 
2. I already created an interface `IBbsArchiveDownloader` to use it both for SSH and SMB downloads and because of that this class can be marked as `sealed` since we don't plan to subclass it.